### PR TITLE
add cafe menu to what to eat response 

### PR DESCRIPTION
**Pull request by Daniel-Pivonka**

### CODE REVIEW

**Review by nathan-weinberg — DISMISSED** [review]
LGTM

**Comment by timflannagan** [comment]
LGTM

File: timbot.py [hunk 1]
```diff
@@ -81,11 +81,20 @@ def run_timbot():
...
 					# what to eat
 					elif 'what' in message and 'eat' in message:
-						sc.api_call("chat.postMessage", channel=channel_name, text="chicken sandwich", as_user=True)
+						# if friday
+						if datetime.datetime.today().weekday() == friday_index_elem:
+							sc.api_call("chat.postMessage", channel=channel_name, text="Its friday enjoy a meal out. Maybe some french toast at pauls?", as_user=True)
+						else:
+							image_url = 'http://cafe.epicureanfeast.com/Clients/8680redhat.jpg'
+							attachments = [{"title": 'Menu', "image_url": image_url}]
+							responce = sc.api_call("chat.postMessage", channel=channel_name, text='Heres the cafe menu', as_user=True, attachments=attachments)
```
Comment (timflannagan) [inline timbot.py]: You don't need to assign to this variable.

Comment (Daniel-Pivonka) [reply timbot.py]: fixed

**Review by nathan-weinberg — APPROVED** [review]
LGTM

**Pull request merged**